modules/developers-guide/examples/javascript/node-gen-table-token.js

```diff
@@ -0,0 +1,17 @@
+// Stargate OSS configuration for locally hosted docker image
+const auth_endpoint = "http://localhost:8081/v1/auth";
+const username = "cassandra";
+const password = "cassandra";
+const stargate_uri = "localhost:8090";
+
+// Set up the authenication
```

Comment:
Same thing, missing a t in authentication

Reply:
fixed in both stargate and astra docs.